Translation/Extractor/FileExtractor.php

```diff
@@ -227,7 +227,7 @@ public function extract()
                         $visitingArgs[] = $ast;
                     } elseif ('twig' === $extension) {
                         $visitingMethod = 'visitTwigFile';
-                        $visitingArgs[] = $this->twig->parse($this->twig->tokenize(file_get_contents($file), (string) $file));
+                        $visitingArgs[] = $this->twig->parse($this->twig->tokenize(file_get_contents($file), (string)$file));
```

Comment:
Please revert this. It violates the code style. 

Also, it is good practice not to make CS changes mixed with a feature. CS changes should be in a separate PR.